.toolversions

```diff
@@ -0,0 +1,25 @@
+cmake="
+  DeclaredVersion='3.6.0'
+  DownloadFileLinux='cmake-3.6.0-Linux-x86_64.tar.gz'
+  DownloadFileOSX='cmake-3.6.0-Darwin-x86_64.tar.gz'
+  DownloadFileWindows32='cmake-3.6.0-win32-x86.zip'
+  DownloadFileWindows64='cmake-3.6.0-win64-x64.zip'
+  DownloadUrl='https://cmake.org/files/v3.6/'
+  LocalToolFolder='Tools/downloads/cmake'
```

Comment:
Agree. Removed, and updated `get_local_tool_folder` to return the default path as `Tools/downloads` under repo root.